diff --git a/node_modules/curlconverter/dist/src/shell/webParser.js b/node_modules/curlconverter/dist/src/shell/webParser.js
index 1b3bab1..b863f3e 100644
--- a/node_modules/curlconverter/dist/src/shell/webParser.js
+++ b/node_modules/curlconverter/dist/src/shell/webParser.js
@@ -5,10 +5,10 @@ import Parser from "web-tree-sitter";
 await Parser.init({
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     locateFile(scriptName, scriptDirectory) {
-        return "/" + scriptName;
+        return "js/" + scriptName;
     },
 });
-const Bash = await Parser.Language.load("/tree-sitter-bash.wasm");
+const Bash = await Parser.Language.load("js/tree-sitter-bash.wasm");
 const parser = new Parser();
 parser.setLanguage(Bash);
 export default parser;
diff --git a/node_modules/curlconverter/src/shell/webParser.ts b/node_modules/curlconverter/src/shell/webParser.ts
index 3222102..e0847ec 100644
--- a/node_modules/curlconverter/src/shell/webParser.ts
+++ b/node_modules/curlconverter/src/shell/webParser.ts
@@ -7,10 +7,10 @@ import Parser from "web-tree-sitter";
 await Parser.init({
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   locateFile(scriptName: string, scriptDirectory: string) {
-    return "/" + scriptName;
+    return "js/" + scriptName;
   },
 });
-const Bash = await Parser.Language.load("/tree-sitter-bash.wasm");
+const Bash = await Parser.Language.load("js/tree-sitter-bash.wasm");
 const parser = new Parser();
 parser.setLanguage(Bash);
 
